task 1/detect.cpp

```diff
@@ -0,0 +1,256 @@
+#include <vector>
+#include <iostream>
+#include <algorithm>
+#include <opencv2/core.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/imgproc.hpp>
+
+// Parameters
+const double MIN_RECT_AREA = 50;
+const double MIN_WH_RATIO = 2;    // Min Ratio of width & height
+const double MAX_WH_RATIO = 10;   // Max Ratio of width & height
+const double MIN_AREA_RATIO = 0.1;// Min Ratio of area & rect
+const double Norm2 = 9;           // The Second Norm (isSameBar)
+const int light_threshold = 229;  // Light threshold (Red 233 Blue 222)
+const int dim_threshold = 255;    // Dim threshold
+const int ENEMY_COLOR = 2;        // For Blue is 0, Red is 2
+const double ANGEL_RANGE = 5;     // Angel threshold of two light bars (Red 5 Blue 2.5)
+const double HEIGHT_RANGE = 25;   // Height threshold of two light bars (Red 30 Blue)
```

Comment:
please use the same coding style for the const.

Reply:
Make it better!!!

Reply:
Make it better!!